scripts/update-cli.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+tagPrefix=''
+repo="https://github.com/getsentry/sentry-cli.git"
+packages=('@sentry/cli')
+
+. $(dirname "$0")/update-package-json.sh
```

Comment:
Sentry-Capacitor doesn't have Sentry CLI as a dependency. Do we want to change that or otherwise we should remove this update script?